.github/workflows/main.yml

```diff
@@ -13,10 +13,10 @@ on:
 jobs:
   lint:
     name: Lint all Markdown files
-    runs-on: ubuntu-20.04
```

Comment:
**suggestion**: I'm usually not a big fan of using latest in CI pipelines, as it is considered a bad practice because of potential unwanted changes.

I would keep a pinned version ideally